integration/create_test.go

```diff
@@ -26,6 +27,57 @@ import (
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
+func TestCreateStressPostgres(t *testing.T) {
+	setup.SkipForPostgresWithReason(t, "https://github.com/FerretDB/FerretDB/issues/1206")
+	setup.SkipForTigrisWithReason(t, "Tigris needs a schema for collection creation")
+
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t) // no providers there, we will create collections concurrently
+	db := collection.Database()
+
+	collNum := 10
+
+	var wg sync.WaitGroup
+	for i := 0; i < collNum; i++ {
+		wg.Add(1)
+
+		go func(i int) {
+			collName := fmt.Sprintf("stress_%d", i)
+			err := db.CreateCollection(ctx, collName)
+			_, errIns := db.Collection(collName).InsertOne(ctx, bson.D{{"_id", "foo"}, {"v", "bar"}})
+
+			wg.Done()
+
+			require.NoError(t, err)
+			require.NoError(t, errIns)
+		}(i)
+	}
+
+	wg.Wait()
+
+	colls, err := db.ListCollectionNames(ctx, bson.D{})
+	require.NoError(t, err)
+
+	// TODO https://github.com/FerretDB/FerretDB/issues/1206
+	// This check fails now. Even though all the collections are created as separate tables in the database,
```

Comment:
Well, it doesn't, because we skip that test for both PostgreSQL and Tigris

Reply:
fixed